datalake/src/main/java/com/sequenceiq/datalake/controller/sdx/SdxController.java

```diff
@@ -265,9 +265,10 @@ public SdxDatabaseBackupResponse backupDatabaseByName(@ResourceName String name,
 
     @Override
     @CheckPermissionByAccount(action = AuthorizationResourceAction.RESTORE_DATALAKE)
-    public SdxDatabaseRestoreResponse restoreDatabaseByName(@ResourceName String name, String backupId, String backupLocation) {
+    public SdxDatabaseRestoreResponse restoreDatabaseByName(@ResourceName String name, String backupId,
```

Comment:
Why are we passing both backupId and restoreId to the restore operation calls? I don't see where we're actually using backupId, although I may just be missing it.

Reply:
we need backup-id to use the exact backup location for database backup. This was there from before.
We need restore-id to poll the status of datalake-dr operation for the restore to end the flow.